.gitignore

```diff
@@ -2,3 +2,4 @@ dist
 .nyc_output
 coverage
 temp
+node_modules/
```

Comment:
I took the liberty to add `node_modules` here, since we probably never want this to be pushed.